meta/labs_program/interview.md

```diff
@@ -0,0 +1,16 @@
+Explain what labs program is.
+
+1. Tell me about your extracurriculars. What does your day look like after school ends? Can you rank them in order of priority?
+2. So you have a bunch of shit going on. Why do you care about this? 
+3. Imagine you are someone walking into your club for the first time. What's that experience like? How does someone find out about your club? What are the biggest problems you foresee running into?
+4. If your club goes really well, what will that look like?
+5. Why do you want to be part of the labs program? Can you spend 5 hours outside of your club a week?
```

Comment:
Can you wrap these lines to 80 characters? Happy to help out with doing this (just grab me in person).

The following parts of our [Markdown style guide](https://github.com/hackedu/meta/blob/master/markdown_style_guide.md) are relevant:

- https://github.com/hackedu/meta/blob/master/markdown_style_guide.md#lists
- https://github.com/hackedu/meta/blob/master/markdown_style_guide.md#basic-conventions